cpp/src/slice2cpp/Gen.cpp

```diff
@@ -2000,17 +2000,16 @@ Slice::Gen::DataDefVisitor::visitModuleStart(const ModulePtr& p)
 }
 
 void
-Slice::Gen::DataDefVisitor::visitModuleEnd(const ModulePtr& p)
+Slice::Gen::DataDefVisitor::visitModuleEnd(const ModulePtr&)
 {
-    if (p->contains<Struct>())
```

Comment:
The net result of this check was not good since these operators apply to structs, classes and exceptions.

Say your class happened to be in the same module as a struct, you'd get these operators for your class too; if no struct, no operator... an unexpected side effect.